.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+<!---
+Provide a general summary of your changes in the Title above
+Include one of these prefixes:
+  fix – Fixes an unexpected problem or unintended behavior
+  feat – Adds a new feature
+  add - Adds a new ECE student
+Example: add : <name> <year> <site>
+-->
+
+### Summary of Changes
+
+<!-- Include a summary of changes -->
+Attach your name, year and site so we can validate your entry 😊
```

Comment:
"Attach your name, year **of graduation** and site so we can validate your entry. 😊"